.github/workflows/ci-mobile-save-for-later.yaml

```
@@ -8,18 +8,21 @@ jobs:
       id-token: write
       contents: read
     steps:
-      - uses: actions/checkout@v3
-      - uses: aws-actions/configure-aws-credentials@v1
+      - uses: actions/checkout@v4
+      - uses: aws-actions/configure-aws-credentials@v2
         with:
           role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
           aws-region: eu-west-1
-      - name: Set up JDK 8
-        uses: actions/setup-java@v3
+      - name: Set up JDK 11
```

Tiny-nit: maybe reduce the size of future diffs for future Java upgrades by removing places where we're repeating the Java version unnecessarily:

```suggestion
      - name: Set up JDK
```